packages/kbn-es/src/utils/docker.ts

```diff
@@ -56,6 +56,8 @@ export interface DockerOptions extends EsClusterExecOptions, BaseOptions {
 }
 
 export interface ServerlessOptions extends EsClusterExecOptions, BaseOptions {
+  /** Publish ES docker container on additional host IP */
+  host?: string;
```

Comment:
Can we add this to the cli `--help` to https://github.com/elastic/kibana/blob/main/packages/kbn-es/src/cli_commands/serverless.ts#L37?